Entities/Common/Building/PlacementCommon.as

```diff
@@ -126,25 +126,40 @@ bool isBuildableAtPos(CBlob@ this, Vec2f p, TileType buildTile, CBlob @blob, boo
 		bool isLadder = false;
 		bool isSpikes = false;
 		bool isDoor = false;
+		bool isPlatform = false;
 		bool isSeed = false;
+
 		if (blob !is null)
 		{
 			const string bname = blob.getName();
 			isLadder = bname == "ladder";
 			isSpikes = bname == "spikes";
 			isDoor = bname == "wooden_door" || bname == "stone_door" || bname == "bridge";
+			isPlatform = bname == "wooden_platform";
 			isSeed = bname == "seed";
-
 		}
 
 		Vec2f middle = p;
 
-		if (!isSeed && !isLadder && (buildSolid || isSpikes || isDoor) && map.getSectorAtPosition(middle, "no build") !is null)
+		s32 x = Maths::Floor(p.x);
+		x /= map.tilesize;
+		s32 y = Maths::Floor(p.y);
+		y /= map.tilesize;
+
+		CBlob@ blobAtPos = map.getBlobAtPosition(Vec2f(p.x, p.y));
```

Comment:
would it not be better to get all blobs at said position in case we have overlapping blobs, it could prevent you from repairing

Reply:
very true, fixed